lab/changelog.md

```diff
@@ -4,6 +4,56 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## Lab v2.5.0 - <TODO: Add the release date>
+
+## What's New
+
+* In case of multiple Cypher queries in the query editor, you will be able to select a single query and run
+  just that one without a need to comment out all the other Cypher queries.
+
+<TODO: Add screenshot here>
+
+* A limit of having one query execution view has been removed and you can have multiple query executions now. It makes
+  life easier especially when you want to compare different Cypher queries side to side in terms of speed or
+  query results.
```

Comment:
```suggestion
* You can now open multiple query executions views and compare execution speed or results of Cypher queries side by side.
```

Reply:
Hahah this is great, my complex 3 lines were replaced with a simple 1-2 liner. :D